openpype/modules/deadline/plugins/publish/collect_pools.py

```diff
@@ -1,5 +1,15 @@
 # -*- coding: utf-8 -*-
-"""Collect Deadline pools. Choose default one from Settings
+"""Collect pools from instance or attributes, from Setting otherwise.
```

Comment:
I feel like there's no need to duplicate this 'docstring' to be both here and on the plugin class.

Reply:
Yeah, you are right, it was before (as docstring were shown in old Publisher - so they were more artist oriented).

I wanted to have it still separate and bring up discussion about `artist` and `dev` docstrings.